test/Sema/implicit_lifetime_dependence.swift

```diff
@@ -18,7 +18,7 @@ struct ImplicitInit2 : ~Escapable, ~Copyable {
   let mbv: BufferView
 }
 
-struct ImplicitInit3 : ~Escapable, ~Copyable { // expected-error{{cannot infer lifetime dependence on implicit initializer, multiple parameters qualifiy as a candidate}}
+struct ImplicitInit3 : ~Escapable, ~Copyable { // expected-error{{cannot infer lifetime dependence on implicit initializer, multiple parameters qualify as a candidate}}
```

Comment:
This change is required in #75020 where you fix the actual message template. We should drop it here.

Reply:
reverted